src/ucp/rndv/proto_rndv.c

```diff
@@ -311,13 +312,35 @@ ucs_status_t ucp_proto_rndv_rts_init(const ucp_proto_init_params_t *init_params)
         .mem_info.sys_dev    = init_params->select_param->sys_dev
     };
 
-    UCP_RMA_PROTO_INIT_CHECK(init_params, UCP_OP_ID_TAG_SEND);
+    return params;
+}
 
-    if (init_params->select_param->dt_class != UCP_DATATYPE_CONTIG) {
```

Comment:
can it be removed already?

Reply:
Do you mean remove the check for datatype inside initialization of RTS protocols?
If so, then the answer is no. Because it is not implemented yet, there are other places with asserts to check the datatype.

Reply:
no, i meant we can't remove it now, but I see it is moved to another place